underdocs-renderer/src/main/kotlin/underdocs/renderer/representation/documentation/ErrorHandling.kt

```diff
@@ -0,0 +1,14 @@
+package underdocs.renderer.representation.documentation
+
+data class ErrorHandlingItem(
+    val actionState: String,
```

Comment:
Minor thing. I'm not quite sure about the name of this property. Fore me, it reads like "action state" instead of "action or state" in its current form. Maybe we don't have to keep both, but can get away with using only "state" or "action", what do you think?

Reply:
Sure, let's use "state" then.

Reply:
Cool!